docs/readme/benchmark.md

```diff
@@ -10,21 +10,21 @@ maxatac benchmark --prediction GM12878_CTCF_chr1.bw --gold_standard GM12878_CTCF
 
 ## Required Arguments
 
-### `--prediction`
+### `-bed, --bed, -bw, --bw, --bigwig`
 
-The input bigwig file of transcription factor binding predictions. This file can also be any bigwig signal track that you want to compare against a gold standard.
+The input file of transcription factor binding predictions (in either BED or bigWig format). This file can also be any BED/bigWig signal track that you want to compare against a gold standard.
```

Comment:
How do you get a BED file at the end of prediction? You should only be getting a bw file. The input flag should --bw here

Reply:
I might totally be wrong, but this is what I was pointing out in the previous screenshot! Please let me know if I am wrong and I am happy to change what I have written.